javascript/javascript1/week3/smartphoneapp.js

```diff
@@ -0,0 +1,80 @@
+const activities = []
+function addActivity(activity, duration){
+    var today = new Date();
+today = today.toLocaleDateString("en-US");
+activities.push({
+        date: today,
+        activity: activity,
+        duration: duration,
+})
+}
+addActivity("Youtube", 30);
+function showStatus(activities){
+    if(activities.length==0){
```

Comment:
Try to avoid using `==`. Always use `===`, even if you need to parse into another data type.

Reply:
done